sharktank/sharktank/kernels/wave/extend_attention.py

```diff
@@ -65,8 +65,8 @@ def get_wave_extend_attention_asm(
         q_extend_shape,
         k_extend_shape,
         v_extend_shape,
-        k_cache_shape,
-        v_cache_shape,
+        kv_cache_1_shape,
```

Comment:
Don't do kv_cache_#.  This is ambiguous. It should be clear what these buffer / shape represents (e.g. the k or v component). Its especially important when invoking for wave.